### PR TITLE
feat: demo geolocation instead of rewrites

### PR DESCRIPTION
**Pull request by orinokai**
Swapped out the original demo for a page that demonstrates geolocation using middleware running in an edge function.

The middleware fetches the users location and rewrite the URL to include a country parameter, which is read by the page template and output as text and a flag image.

### CODE REVIEW

**Review by sarahetter — APPROVED** [review]
LGTM

File: lib/countries.js [hunk 1]
```diff
@@ -0,0 +1,245 @@
+export default [
+  { name: 'Afghanistan', code: 'AF' },
+  { name: 'Åland Islands', code: 'AX' },
+  { name: 'Albania', code: 'AL' },
+  { name: 'Algeria', code: 'DZ' },
+  { name: 'American Samoa', code: 'AS' },
+  { name: 'AndorrA', code: 'AD' },
```
[the remark this answers is not in the document]
Comment (orinokai) [reply lib/countries.js]: yes, well spotted! would you mind re-approving? @sarahetter

**Review by sarahetter — APPROVED** [review]

File: pages/index.js [hunk 2]
```diff
@@ -1,31 +1,43 @@
...
+              <span className={styles.flag}>
+                <Image
+                  alt={`${country} flag`}
+                  src={`https://flagcdn.com/96x72/${country.toLowerCase()}.png`}
+                  width={32}
+                  height={24}
+                />
+              </span>
+              <span className={styles.welcome}>
+                Welcome from {countries.find((c) => c.code === country).name}
```
Comment (ascorbic) [inline pages/index.js:35]: The silly thing is that the geo object that we pass an Edge Function already includes the country name, but Next.js doesn't expose it!

Comment (orinokai) [reply pages/index.js:35]: Gah!

**Pull request merged**